playground/cdk/CdkSample.AppHost/Program.cs

```diff
@@ -45,20 +51,24 @@
     .AddQueue("queue1",
         (_, construct, queue) =>
         {
-            queue.Properties.MaxDeliveryCount = 5;
-            queue.Properties.LockDuration = TimeSpan.FromMinutes(5);
+            queue.MaxDeliveryCount = 5;
+            queue.LockDuration = new StringLiteral("PT5M");
+            // TODO: this should be
+            // queue.LockDuration = TimeSpan.FromMinutes(5);
```

Comment:
@tg-msft

Reply:
It's a known issue - SB uses ISO8601 durations _(that are no longer allowed in Azure, save for recurrences or human-centric time intervals, because no languages support them well)_ and we'll need to carry along a format to serialize TimeSpans.